sw.js

```diff
@@ -0,0 +1,38 @@
+---
+layout: null
+---
+
+importScripts( '{{ site.baseurl }}/cache-polyfill.js' );
+
+var filesToCache = [
+  // root
+  '{{ site.baseurl }}/',
+  '{{ site.baseurl }}/index.html',
+  // css
+  '{{ site.baseurl }}/assets/css/main.css',
+  '{{ site.baseurl }}/assets/css/normalize.css',
+  '{{ site.baseurl }}/assets/css/syntax.css',
+  // images
+  '{{ site.baseurl }}/assets/img/octocat.png',
+  // pages
+  '{{ site.baseurl }}/example_page/',
```

Comment:
I can't tell you how long I spent trying to figure out the sytax for this. That seems to work.